### PR TITLE
Added cooler counter

### PR DESCRIPTION
**Pull request by MarkusTarn**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" type="image/svg+xml" sizes="any" href="%PUBLIC_URL%/favicon.svg">
+  <link href="https://fonts.googleapis.com/css2?family=Margarine&display=swap" rel="stylesheet">
```
Comment (stivka) [inline public/index.html:7]: lets maintain a clean index.html and keep styling configurations in theme.js

Comment (stivka) [inline public/index.html:7]: lets maintain a clean index.html and keep styling configurations in theme.js

File: public/index.html [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" type="image/svg+xml" sizes="any" href="%PUBLIC_URL%/favicon.svg">
```
Comment (stivka) [inline public/index.html:6]: lets maintain a clean index.html and keep styling configurations in theme.js

Comment (meelikalukner) [reply public/index.html:6]: yes

**Review by stivka — APPROVED** [review]

**Pull request merged**